seed/static/seed/js/controllers/mapping_controller.js

```diff
@@ -720,23 +720,26 @@ angular.module('BE.seed.controller.mapping', [])
       var display_cached_column_mappings = function () {
         var cached_mappings = JSON.parse($scope.import_file.cached_mapped_columns);
         _.forEach($scope.mappings, function (col) {
-          var cached_col = _.find(cached_mappings, {from_field: col.name, to_table_name: col.suggestion_table_name})
-          col.suggestion_column_name = cached_col.to_field;
-          col.from_units = cached_col.from_units;
-
+          var cached_col = _.find(cached_mappings, {from_field: col.name})
+          if (cached_col == null) {
```

Comment:
I think the only possible values of `cached_col` is an object for the column or `undefined`, so I think this always enters the `else` logic.

I think it's likely that the only fixed needed was to remove the "to_table_name" key and value from the `_.find`, but I haven't look at this more closely, so let me know if I misunderstood anything.

Reply:
Hey, @adrian-lara thank you for the review!  The main reason I did the misc formatting push was to try and clear the License Compliance however Alex told the that it sometimes just doesn't pass.  I will remove the console log and also check out the null/undefined logic now that it should contain values